apps/web/src/store/modules/settings/actions.ts

```diff
@@ -1,5 +1,21 @@
 import type { SetItemRequest } from './type';
 
-export const setItem = ({ commit }, item: SetItemRequest): void => {
-    commit('setItem', item);
+export const setItem = ({ commit, rootState }, item: SetItemRequest): void => {
+    const userId = rootState?.user?.userId;
+    if (userId) commit('setUserSetting', { item });
+};
+
+export const initSettings = ({ commit, rootState }): void => {
+    const userId = rootState?.user?.userId;
+    if (!window.localStorage.getItem('spaceConnector/accessToken')) return;
```

Comment:
suggestion: I think it would be more maintainable to use `isSessionExpired` flag instead of access token storage key.